Docker/docker-compose.yml

```diff
@@ -2,9 +2,20 @@ version: "2"
 
```

Comment:
line 1: @toonsevrin Use version 3

Reply:
@peterwillcn I don't think we can reuse the config because then all plugins will be loaded to both apps. Let's keep the seperate configs.